primo/utils/setup_logger.py

```diff
@@ -87,3 +87,6 @@ def setup_logger(
         datefmt=LOGGER_DATE,
         handlers=handlers,
     )
+
```

Comment:
Please replace the lines starting with `logging.basicConfig(...` with the following lines
```python
if log_level in [0, 1, 2]:
        logger_format = "primo: %(levelname)s: %(message)s"
        logger_date = None

    else:
        logger_format = "%(asctime)s - %(name)s:%(lineno)d - %(levelname)s - %(message)s"
        logger_date = "%d-%b-%y %H:%M:%S"

    logging.basicConfig(
        level=supported_log_levels[log_level],
        format=logger_format,
        datefmt=logger_date,
        handlers=handlers,
    )

    logger = logging.getLogger("gurobipy")
    logger.propagate = False

    logger = logging.getLogger("pyomo.contrib.appsi.solvers.highs")
    logger.propagate = False
```

And, remove `LOGGER_FORMAT` and `LOGGER_DATE` variables located at the top of the file.